crane_plus_examples/src/pick_and_place.cpp

```diff
@@ -58,6 +58,10 @@ int main(int argc, char ** argv)
   double GRIPPER_OPEN = to_radians(-30);
   double GRIPPER_CLOSE = to_radians(10);
 
+  // Set goal tolerance for the target pose to improve IK
```

Comment:
IKの成功率まで含めるといいかと思います。
```suggestion
  // Set goal tolerances to improve IK success rate
```

Reply:
修正しました！